dev/source/docs/ros-timesync.rst

```diff
@@ -28,17 +28,7 @@ ArduPilot Configuration
 
 The flight controller's time is set from the GPS by default but can be set from one of three sources(GPS, MAVLINK, onboard HW clock) using the `BRD_RTC_TYPES <https://ardupilot.org/copter/docs/parameters.html?highlight=parameters#brd-rtc-types-allowed-sources-of-rtc-time>`__ parameter as a bit mask for the time syncronisation source. When using mavros set BRD_RTC_TYPES = 2 to only allow the "MAVLINK_SYSTEM_TIME" source which uses the `SYSTEM_TIME <https://mavlink.io/en/messages/common.html#SYSTEM_TIME>`__ MAVLINK message.
 
-For Rover or Plane, the SCHED_LOOP_RATE parameter should be increased to 200 because mavros ignores TIMESYNC messages if the round trip time is more than 10ms.  It may also be necessary to increase the update rate of the `gcs related tasks in the scheduler <https://github.com/ArduPilot/ardupilot/blob/master/Rover/Rover.cpp#L67>`__.
-
-For rover, open `Rover.cpp <https://github.com/ArduPilot/ardupilot/blob/master/Rover/Rover.cpp>`__ and within the scheduler_tasks[] array ensure the gcs entries have their update rate set to 200 like below:
-
-::
-
-    SCHED_TASK(gcs_retry_deferred,    200,    500),
-    SCHED_TASK(gcs_update,            200,    500),
-    SCHED_TASK(gcs_data_stream_send,  200,   1000),
-
-After making this change you will need to compile rover and upload the new binary to the flight controller.  Hopefully this will not be necessary in future versions of Plane or Rover.
+For ArduPilot, the SCHED_LOOP_RATE parameter should be increased because mavros ignores TIMESYNC messages if the round trip time is more than 10ms.
```

Comment:
I don't think it should be raise on copter, no  ?

Reply:
I've only ever seen it on Rover, so will put in a note that it will _generally_ only occur on rover